features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/postgresql.sql

```diff
@@ -1187,7 +1187,7 @@ CREATE TABLE IDN_FED_AUTH_SESSION_MAPPING (
 	AUTHENTICATOR_ID VARCHAR(255),
 	PROTOCOL_TYPE VARCHAR(255),
 	TIME_CREATED TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-	TENANT_ID INTEGER NOT NULL DEFAULT 0
+	TENANT_ID INTEGER NOT NULL DEFAULT 0,
```

Comment:
is this change introduced recently?

Reply:
yes, this was added by https://github.com/wso2/carbon-identity-framework/pull/4035